src/ResourceManager/Insights/Commands.Insights/LogProfiles/GetAzureRmLogProfileCommand.cs

```diff
@@ -45,17 +45,17 @@ protected override void ProcessRecordInternal()
             var result = new PSLogProfileCollection();
             if (string.IsNullOrWhiteSpace(this.Name))
             {
-                IEnumerable<LogProfileResource> resultList = this.InsightsManagementClient.LogProfiles.ListAsync(cancellationToken: CancellationToken.None).Result;
+                IEnumerable<LogProfileResource> resultList = this.MonitorManagementClient.LogProfiles.ListAsync(cancellationToken: CancellationToken.None).Result;
 
                 result.AddRange(resultList.Select(x => new PSLogProfile(logProfile: x)));
             }
             else
             {
-                LogProfileResource logProfile = this.InsightsManagementClient.LogProfiles.GetAsync(logProfileName: this.Name, cancellationToken: CancellationToken.None).Result;
+                LogProfileResource logProfile = this.MonitorManagementClient.LogProfiles.GetAsync(logProfileName: this.Name, cancellationToken: CancellationToken.None).Result;
                 result.Add(new PSLogProfile(logProfile: logProfile));
             }
 
-            WriteObject(result);
+            WriteObject(result, enumerateCollection: true);
```

Comment:
PsLogProfileCollection is actually a List<PSLogProfile>. Here again I think Vinicius is using the PS class to modify the display of this results.